### ISSUE TITLE
Missing unused argument warning for short form function

### ISSUE THREAD

**Issue by fredrikekre** [julia-vscode/StaticLint.jl#312]
```
g(x) = 1
```
doesn't warn for unused `x` like
```
function f(x)
    return 1
end
```
